test/basics.js

```diff
@@ -0,0 +1,21 @@
+const chai = require('chai');
+const expect = chai.expect;
+const testDummy = require('../src/TestDummy.js')
+
+describe( 'Are tests running', () => {
+
+  context ('When trying to run a basic test file', () => {
+      describe(' and trying to see if describe works', () => {
+        it('Should tell us tests are running', () => {
+          expect(true).to.eql(true)
```

Comment:
`eql` is for deep equality comparisons (e.g., if you're comparing an array of elements to another array of elements). Use `equal` here.